movement/make/Makefile

```diff
@@ -87,7 +87,6 @@ SRCS += \
   ../watch_faces/complication/ratemeter_face.c \
   ../watch_faces/complication/interval_face.c \
   ../watch_faces/complication/rpn_calculator_alt_face.c \
-  ../watch_faces/complication/stock_stopwatch_face.c \
```

Comment:
Before merging, I will refactor this face to use the HPT and add it back to the project.